### DISCUSSION TITLE
controller.masteryService.registerMasteryData is unable to push data to LOCATION_PARENT_ICA_FACILITY

### DISCUSSION THREAD

**Discussion by Burn-the-w1tch** [thepeacockproject/Peacock#513] [Q&A]
**Describe the bug**
controller.masteryService.registerMasteryData is unable to push data to LOCATION_PARENT_ICA_FACILITY for some reason. I have forced the tab to show in the destination menu and it is not loading, so this is Peacock side, not client side.

**To reproduce**
```
Facilitymasterydata = {
        "LocationId": "LOCATION_PARENT_ICA_FACILITY",
        "GameVersions": ["h3"],
        "MaxLevel": 5,
        "Drops": [ {
            "Id": "PROP_TOOL_COIN",
            "Level": 1
        },
        {
            "Id": "TOKEN_FIBERWIRE",
            "Level": 2
        },
        {
            "Id": "TPROP_TOOL_LOCK_PICK",
            "Level": 3
        },
        {
            "Id": "FIREARMS_HERO_PISTOL_TACTICAL_ICA_19",
            "Level": 4
        },
        {
            "Id": "TOKEN_OUTFIT_LEGACY_HERO_SIGNATURESUITANDGLOVES",
            "Level": 5
        },
    
    ]
    }
    controller.masteryService.registerMasteryData(Facilitymasterydata);
```

**Additional context**
![image](https://github.com/thepeacockproject/Peacock/assets/64156611/ddd74bdd-e6ef-4916-8671-9d478e4c8478)

**Comment by suanjiansalt** [comment]
Guessing you gave the entire code, but it seems that the format of `module.exports = function mastery(controller) { ... }` is required as the main function to run the plugin.

![image](https://github.com/user-attachments/assets/a5e02731-4be2-4110-ae86-dcd9513b339e)

By the way, you typed an extra T for level 3, which makes the lockpick missing.
